docs/src/man/comparisons.md

```diff
@@ -20,40 +20,38 @@ df = pd.DataFrame(
 
 Note that pandas allows row indexing by labels. For DataFrames.jl, you can achieve the same result via row filtering as shown below.
 
-How to access data:
+### Accessing data
 
-| Operation                  | Example                     | pandas                 | DataFrames.jl                                              |
-|:---------------------------|:----------------------------|:-----------------------|:-----------------------------------------------------------|
-| Cell indexing by location  | Cell at row 2, column 2     | `df.iloc[1, 1]`        | `df[2, 2]` (1-based index)*                                 |
-| Row slicing by location    | Rows 2 and 3                | `df.iloc[1:3]`         | `df[2:3, :]`                                               |
-| Column slicing by location | Column 2 and after          | `df.iloc[:, 1:]`       | `df[:, 2:end]`*                                            |
-| Row indexing by label      | Row 'c'                     | `df.loc['c']`          | `df[df.id .== 'c', :]`                                     |
-| Column indexing by label   | Column 'x'                  | `df.loc[:, 'x']`       | `df[:, :x]`                                                |
-| Column slicing by label    | Columns 'x' and 'z'         | `df.loc[:, ['x','z']]` | `df[:, [:x, :z]]`  or `select(df, [:x, :y])`               |
-|                            | Columns between 'x' and 'z' | `df.loc[:, 'x':'z']`   | `df[:, Between(:x, :z)]`  of `select(df, Between(:x, :y))` |
-| Mixed indexing             | Cell at row 'c', column 2   | `df.loc['c'][1]`       | `df[df.id .== 'c', 2]`*                                    |
+| Operation                  | Example                     | pandas                 | DataFrames.jl            |
```

Comment:
maybe instead of "Example" column just add this in "Operation" column in parenthesis (this way we can flexibly decide where it makes sense to add a comment and when we can omit it). What do you think?

Reply:
Worth trying, yes!

Reply:
I'm now more inclined to delete the `Example` column since the audience would be pandas users and they should know it really well already.

Reply:
OK - let us delete this column

Reply:
Column is still here?

Reply:
👻 ! It's gone now 😄